docs/concepts/pdp/configuration.mdx

```diff
@@ -315,6 +329,70 @@ Default: `False`
 
 Enable FactDB for this PDP. For more information, see the [FactDB documentation](/concepts/pdp/factdb).
 
+#### PDP_CACHE_STORE
+
+Default: `none`
+
+The cache storage type. Options include: none, in-memory, and redis.
+For more information, see the [PDP Cache documentation](/concepts/pdp/overview#pdp-caching-mechanism).
+
+_Added in PDP v0.9.0_
+
+#### PDP_CACHE_TTL
```

Comment:
I think it's PDP_CACHE__TTL ( with double `_` because it's nested )

Reply:
Same, it was changed